backend/danswer/document_index/vespa/app_config/services.xml

```diff
@@ -2,7 +2,12 @@
 <services version="1.0">
     <container id="default" version="1.0">
         <document-api/>
-        <search/>
+        <search>
+            <threadpool>
+                <threads>SEARCH_THREAD_NUMBER</threads>
```

Comment:
There seem to be several thread settings for Vespa.  Is this the setting we want to change? I found a different set here and persearch seems to be relevant ... but applies to the content node, not the container?

https://docs.vespa.ai/en/reference/services-content.html#requestthreads